src/JungleLvl/private/ground.cpp

```diff
@@ -14,6 +14,6 @@ Ground::Ground() : CRAWL::GroundClaster("JungelGroud") {
 }
 
 unsigned int Ground::itemsCount() const {
-    return 3;
+    return 5;
```

Comment:
you need to find bug in render function of the Ground object. but not increment ground items count.
```suggestion
    return 3;
```